### PR TITLE
Update documentation with android background notifications

### PR DESCRIPTION
**Pull request by spike1292**
Add improvements task list
Add sdk documentation 
Add documentation how to enable android background notifications

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,100 +4,94 @@
...
+    // Toggles between the development and production app credentials
+    // Before submitting your application to an app store set to true
+    inProduction: false,
+    productionAppKey: "Your Production App Key",
+    productionAppSecret: "Your Production Secret"
+};
 ```
 
-#### push-notification.ts
+> app/push-notification.ts
```
Comment (code0wl) [inline README.md]: good

File: README.md [hunk 2]
```diff
@@ -116,8 +110,53 @@ class YourApplicationName extends android.app.Application {
...
+}
+```
+
+## Usage Example
+``` typescript
+NsUrbanairship.getInstance().isOptIn(); // return a boolean if the user has registered for notifications
+NsUrbanairship.getInstance().unRegisterUser(); // un-registers the user from receiving notifications
+```
+
+## Improvements
```
Comment (code0wl) [inline README.md]: Move to changelog.md

Comment (spike1292) [inline README.md]: these are more a kind of issues that need to be fixed

Comment (spike1292) [inline README.md]: but adding a changeling is not a bad idea 😉

Comment (code0wl) [inline README.md]: Ah, this is probably something for ROADMAP.md then.

Comment (spike1292) [inline README.md]: or GitHub issues? discuss tomorrow?

**Pull request merged**